content/2017/03/30/safe-get.rst

```diff
@@ -131,161 +135,101 @@ In demo code, people often write it like this:
-      // if it's a string, but we can get a function, we should
-      @if function-exists('get-function') {
-        @return get-function($function);
-      }
-
-      // if it's a string, and we can't get a function, return the string
-      @return $function;
-    }
+After a long conversation with Chris Eppstein,
+one of the Sass language designers,
+it's clear that the change rests
+entirely in the hands of users.
+Most toolkits can continue to use ``call``
+as they always have,
+but users upgrading to Sass 3.5+
+should begin to capture functions
+before passing them anywhere outside the local context.
 
-    // if it's not a string or a function, we know there's a problem
-    @error 'Invalid function name, [#{$type}] `#{$function}` must be a function or string';
-  }
+OddBird's `Sass Accoutrement`_ tools,
+for example,
+allow users to `pass in an arbitrary function`_.
+Once users upgrade to Sass 3.5,
+they should be sure to ``get`` the function
+before passing it in.
+Meanwhile,
+our tools will continue to use ``call`` internally,
+without any changes.
 
-This ``safe-get-function`` accepts one argument,
-either a string or a first-class function,
-and returns the proper value
-(also a string or a function)
-for the version of Sass being used.
 
-
-Safe ``call``
+One Exception
 -------------
 
-I also wrote a very small
-``safe-call()`` wrapper function
-that passes all function-calls
-through our ``safe-get-function()``
-before calling them.
-
-.. code:: scss
-
-  @function safe-call(
-    $name,
-    $args...
-  ) {
-    $name: safe-get-function($name);
-    @return call($name, $args...);
-  }
+There is one exception,
+where I use ``call`` internally,
+with known local functions in a loop.
+The purpose of ``call`` in this case
+is not to accept arbitrary functions from the user,
+but to DRY our code with a loop of local functions.
 
-This function `accepts the same arguments`_
-required by Sass's internal ``call()`` function,
-a name (or first-class function),
-and arguments to pass-through when the function is called.
-You can use it right away like this:
+To handle that,
+we use a few lines of code
+to make sure we ``get`` the function
+in newer versions of Sass,
+without breaking older versions:
 
 .. code:: scss
 
-  $result: safe-call('span', 3);
-
-And that should continue to work just fine
-when you upgrade to Sass 3.5 or later.
+  @each $key, $value in $config {
+    $function: 'susy-normalize-#{$key}';
 
-.. _accepts the same arguments: http://sass-lang.com/documentation/Sass/Script/Functions.html#call-instance_method
+    @if function-exists('get-function') {
+      $function: get-function($function);
+    }
 
+    $result: call($function, $value);
+  }
 
-Ship it!
---------
+This is basically identical to `Kaelig's solution`_,
+which initially inspired my post.
+It should work on all versions of Sass,
+but **should only be used internally,
+calling local functions**
+(e.g. know functions defined in the same partial).
```

Comment:
Are `know functions` a thing? Or should this be `known functions`? I only ask because I changed this once before, and you've reintroduced it. So either you intended it this way, or it's an easy typo to make. :)